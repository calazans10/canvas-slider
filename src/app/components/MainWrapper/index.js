import PropTypes from 'prop-types';
import Wrapper from './style';

function MainWrapper({ children }) {
  return <Wrapper>{children}</Wrapper>;
}

MainWrapper.propTypes = {
  children: PropTypes.node.isRequired,
};

export default MainWrapper;
