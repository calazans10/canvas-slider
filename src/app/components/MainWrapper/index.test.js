import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MainWrapper from '.';

describe('MainWrapper', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(
      <MainWrapper>
        <p>Lorem Ipsum</p>
      </MainWrapper>
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
