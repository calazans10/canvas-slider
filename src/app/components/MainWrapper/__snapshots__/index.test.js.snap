// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainWrapper renders without crashing 1`] = `
<styled.div>
  <p>
    Lorem Ipsum
  </p>
</styled.div>
`;
